kits/peer-demo-showcase/apps/community-showcase/desktop.html

```
@@ -0,0 +1,358 @@
+<a class="text-primary dark:text-primary-fixed border-b-2 border-primary pb-1" href="#">Dashboard</a>
+<a class="text-on-surface-variant hover:text-on-surface transition-colors hover:bg-surface-bright/50 dark:hover:bg-surface-container-highest/50 rounded-lg px-2 py-1" href="#">Explore</a>
+<a class="text-on-surface-variant hover:text-on-surface transition-colors hover:bg-surface-bright/50 dark:hover:bg-surface-container-highest/50 rounded-lg px-2 py-1" href="#">Sponsors</a>
+<a class="text-on-surface-variant hover:text-on-surface transition-colors hover:bg-surface-bright/50 dark:hover:bg-surface-container-highest/50 rounded-lg px-2 py-1" href="#">Community</a>
+</div>
+</div>
+<div class="flex items-center gap-4">
+<div class="relative hidden sm:block">
+<input class="bg-surface-container-low border border-outline-variant rounded-full py-1.5 pl-4 pr-10 text-sm focus:outline-none focus:border-primary text-on-surface w-48" placeholder="Search..." type="text"/>
+<span class="material-symbols-outlined absolute right-3 top-1/2 -translate-y-1/2 text-on-surface-variant text-sm">search</span>
+</div>
+<button class="bg-primary text-on-primary px-4 py-2 rounded-full font-semibold hover:bg-primary-dim transition-colors flex items-center gap-2">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)